node/test/Test/Pos/CborSpec.hs

```diff
@@ -223,11 +232,12 @@ roundtripProperty (input :: a) = ((deserialize . serialize $ input) :: a) === in
 -- without breaking anything. This should work with every time which adopted
 -- the schema of having at least one constructor of the form:
 -- .... | Unknown Word8 ByteString
-extensionProperty :: forall a. (Arbitrary a, Eq a, Show a, Bi a) => Property
-extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
+extensionPropertyOn :: forall a b. (Eq a, Show a, Bi a, Arbitrary b, Eq b, Show b, Bi b)
+                    => Property
+extensionPropertyOn = forAll @b (arbitrary :: Gen b) $ \input ->
```

Comment:
Sorry, probably not fully awake yet ☕️ . `input` is never passed from outside but always generated (see the invocation of this properly, for example [here](https://github.com/input-output-hk/cardano-sl/blob/3e852def50c5b2d45e5ea1e0978a8f4802177066/node/test/Test/Pos/CborSpec.hs#L420)). Do you have anything else in mind that I'm missing?